### PR TITLE
add post following up with development of learners meetup

### PR DESCRIPTION
**Pull request by xMartin**

### CODE REVIEW

File: _posts/2013-06-06-how-the-learners-meetup-is-doing-in-berlin.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "How is the Learners Meetup doing in Berlin? Pretty good!"
+description: ""
+category:
+tags: ['beginners']
```
Comment (gnunicorn) [inline _posts/2013-06-06-how-the-learners-meetup-is-doing-in-berlin.md]: this is berlin-specific. so adding "berlin" to the tags to make it show up in the berlin-json (and therefore in the berlin-website) would be good, too

Comment (xMartin) [inline _posts/2013-06-06-how-the-learners-meetup-is-doing-in-berlin.md]: Ok, cool. I copied from @starkcoffee's post so maybe add it there, too?

Comment (gnunicorn) [inline _posts/2013-06-06-how-the-learners-meetup-is-doing-in-berlin.md]: sounds good.

**Comment by gnunicorn** [comment]
Just that little change and feel free to merge. 

And thanks for the informative post!

**Pull request merged**